### DISCUSSION TITLE
VOICE ACCESS IN THE BOT

### DISCUSSION THREAD

**Discussion by Minty-cyber** [python-telegram-bot/python-telegram-bot#3989] [Q&A]
Can I use a telegram bot to enable voice chat between the bot and the user?

**Comment by Bibo-Joshi** [comment]
Wiki of python-telegram-bot - Category FAQ
[Can you add [feature] to PTB? Can I do [thing] with my bot?](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#can-you-add-feature-to-ptb-can-i-do-thing-with-my-bot)

No :)
